Classes/Helper/DataCiteXml.php

```diff
@@ -46,6 +46,10 @@ public static function convertFromMetsXml($metsXml)
         foreach($metsCreator as $creator)
         {
             $creator->registerXPathNamespace('mods', 'http://www.loc.gov/mods/v3');
+            $role = $creator->xpath(".//mods:roleTerm[@type='code']");
```

Comment:
The creators element is always empty:

```
<?xml version="1.0" encoding="UTF-8"?>
<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4/metadata.xsd">
  <identifier identifierType="DOI">doi:foo/123</identifier>
  <creators/>
  <titles>
    <title>Test Dissertation</title>
  </titles>
  <publisher>TU Dresden</publisher>
  <publicationYear>2017</publicationYear>
  <subjects>
    <subject>xxx</subject>
  </subjects>
  <language>de</language>
  <resourceType resourceTypeGeneral="Text">doctoral_thesis</resourceType>
</resource>

```

Reply:
There was a wrong condition for authors/composers. See commit [5245bce](https://github.com/kitodo/kitodo-publication/pull/70/commits/5245bce1717db1620865b52d60aa98cfeee2ca41).